templates/cube/index.html

```diff
@@ -6,27 +6,23 @@
 
 {% block content %}
 
-<section class="p-takeover--dark">
+<section class="p-strip--square-suru has-cube is-deep is-slanted--bottom-right">
   <div class="row u-equal-height">
-    <div class="col-7">
+    <div class="col-6">
       <h1>You&rsquo;re a professional.</br>Prove it with CUBE.</h1>
       <p>Certified Ubuntu Engineer (CUBE) designation is Canonical&rsquo;s professional endorsement indicating mastery of Ubuntu.</p>
       <p>CUBE candidates must earn 15 microcertifications across relevant subject areas.</p>
       <p>
         <a href="/cube/microcerts" class="p-button--positive">View MicroCerts</a>
       </p>
     </div>
-    <div class="col-5 u-hide--small u-vertically-center u-align--center">
-      {{
-        image(
-        url="https://assets.ubuntu.com/v1/22f88d02-apps.svg",
-        alt="",
-        width="290",
-        height="170",
-        hi_def=True,
-        loading="auto",
-        ) | safe
-      }}
+    <div class="col-5 col-start-large-8 u-hide--small p-cube-animation">
+      <div class="p-cube-animation__cube cube-1"></div>
```

Comment:
That would make these a single class of `p-cube-animation__cube--1`

Reply:
Thanks @anthonydillon, made the suggested changes